python_files/generate_supplementary_plots.py

```diff
@@ -13,6 +23,82 @@
 import seaborn as sns
 
 
+def stitch_and_annotate_padded_img(image_data: xr.DataArray, padding: int = 25,
+                                   font_size: int = 100):
+    """Stitch an image with (x, y, z) dimensions, and annotate each image with labels contained
```

Comment:
Is the `x` dimension the channel `c`?

Reply:
Yes, I'll rename the dimension variables so its clear.